### ISSUE TITLE
Add support for RSS 2.0

### ISSUE THREAD

**Issue by jaredcwhite** [bridgetownrb/bridgetown-feed#5]
The reason this plugin uses Atom is for historical reasons (it started as a fork of the jekyll-feed plugin). I don't have a particular dog in the now-ancient Atom vs. RSS fight, but I do like the idea of supporting RSS 2.0. It's really not much different from the outside looking in. Honestly I'd just love to refactor the whole thing and use a declarative component-based rendering pipeline where the exact XML tag output can easily change depending on the renderer.

**Comment by jaredcwhite** [comment]
**Update:** I looked into this some more, and using Ruby's RSS gem we get a nice declarative builder DSL: https://github.com/ruby/rss

So no more Liquid file required. My plan at this point is to switch entirely to RSS 2.0 in order to support #8 and also just simplify the code.

**Comment by michebble** [comment]
@jaredcwhite 
I was wondering if this is still in progress. I recently migrated to Bridgetown and lost RSS 2.0 in the process, I was looking at rolling my own before I noticed this plugin and this issue. Though I noticed the rss-next branch is stale.

Could I possibly help with its development?

**Comment by jaredcwhite** [comment]
@michebble Yeah it's just on hold while I focus on core features for upcoming Bridgetown 2.0. This is a priority after that. I am using it on one of my sites in production, so feel free to load this branch in your gemfile and try it out. Happy to accept any PRs or issues filed as you run into potential problems. Hopefully it's in decent shape though.

**Comment by pablojimeno** [comment]
Hi @jaredcwhite,

I’ve been using the rss-next branch in a couple of projects and really appreciate the direction you’ve taken with the RSS 2.0 work.

While integrating it, I noticed a few small things that might help move it closer to merge-ready (some dead code cleanup, minor test fixes, and a bit of alignment with Bridgetown 2.x).

Would you be open to a PR building on top of rss-next to help move it forward? Happy to keep the scope focused and incremental.

Thanks again for the work on this, and Bridgetown in general.

**Comment by jaredcwhite** [comment]
Hey @pablojimeno, thanks for reaching out and apologies I didn't get back to you sooner. Been working furiously on the projects Bridgetown had committed to for the Gem Fellowship grant, but now that that batch of work is winding down I can breathe a little.

Yeah, if you still have some bandwidth and would like to collab on this, let me know! Because so many people are using this plugin as-is and I wouldn't want to break that, I'm wondering if we should create a new plugin for the new RSS flavor (like `bridgetown_rss`)?
